LabExT/Movement/Stages/Stage3DSmarAct.py

```diff
@@ -655,7 +638,7 @@ def get_current_position(self) -> list:
 
     @assert_driver_loaded
     @assert_stage_connected
-    def move_relative(self, x, y):
+    def move_relative(self, x, y, wait_for_stopping: bool = True):
```

Comment:
I think the lift_stage() and lower_stage() need the the wait_for_stopping flag as well put to True as default

Reply:
Yep, you're right. I have adjusted the method `lower_stage` and `lift_stage`. These now call the `move_relative` method (with `wait_for_stopping` flag). I added tests for this.